src/tools/db-upgrade/DbUpgrader.cpp

```diff
@@ -83,7 +86,7 @@ Status UpgraderSpace::initSpace(const std::string& sId) {
 
   // Use readonly rocksdb
   readEngine_.reset(new nebula::kvstore::RocksEngine(
-      spaceId_, spaceVidLen_, srcPath_, "", nullptr, nullptr, true));
+      spaceId_, spaceVidLen_, srcPath_, "", nullptr, nullptr, false));
```

Comment:
why change?

Reply:
This rocksdb is originally a read-only

Reply:
Because upgrade to 3.0 just need append new data by ingest sst file and write a `dataVersionKey` to identity the data encode version. So there is no need to write a new rocksdb, just upgrade in place

Reply:
ok

Reply:
Confused about `readEngine_.reset` a writeable one

Reply:
`read` means source and `write` means destnation